### ISSUE TITLE
OSX: Attempting to open Preferences causes a crash

### ISSUE THREAD

**Issue by dangercrow** [Marginal/EDMarketConnector#281]
EDMC Version: 2.4.8.0 (2.4.8.0)
OSX version: 10.12.6 (16G29)
Running on a MBP Mid 2015.

I start the app up, see this:
![image](https://user-images.githubusercontent.com/2048203/34641830-e399eb7c-f301-11e7-839a-e976bd02eb03.png)

Try to open preferences, and the app crashes. No error message or anything, it just dies.

**Comment by Marginal** [comment]
Are you running the app on the same machine as the game?

**Comment by dangercrow** [comment]
Yes I am

**Comment by Marginal** [comment]
Please:
- Run the Terminal app, found in Applications -> Utilities
- Copy the following and paste into the Terminal app window:
   `ls "$HOME/Library/Application Support/Frontier Developments/Elite Dangerous/"; defaults read uk.org.marginal.edmarketconnector journaldir`
- Press Enter.
- Copy the entire contents of the Terminal window (⌘A ⌘C) and paste it in reply here.

**Comment by dangercrow** [comment]
```
$ ls "$HOME/Library/Application Support/Frontier Developments/Elite Dangerous/"
```
```
256084		  GalacticPoliticsPowerBases.cache  Journal.180102172042.01.log  Options
400103		  GalacticPoliticsPowers.cache	    Journal.180104213106.01.log  TelemetryCache.log
AppNetCfg.xml	  Journal.171220202505.01.log	    Journal.180106160623.01.log
CommanderHistory  Journal.171221143214.01.log	    Logs
```

```
$ defaults read uk.org.marginal.edmarketconnector journaldir
```

```
2018-01-06 23:02:56.879 defaults[66216:5810272] 
The domain/default pair of (uk.org.marginal.edmarketconnector, journaldir) does not exist
```

**Comment by Marginal** [comment]
Will be fixed in the next release. In the meantime please downgrade to 2.47.

**Comment by dangercrow** [comment]
![image](https://user-images.githubusercontent.com/2048203/34648247-d0a98b84-f38e-11e7-9674-9967e97e8827.png)
Fix confirmed (running from source @ e791edd)

**Comment by Marginal** [comment]
Great. Thanks.
